server/services/visitSessionsService.ts

```diff
@@ -251,6 +251,24 @@ export default class VisitSessionsService {
     return this.buildVisitInformation(visit)
   }
 
+  async getUpcomingVisitsCancelledAndBooked({
+    username,
+    offenderNo,
+  }: {
+    username: string
+    offenderNo: string
+  }): Promise<VisitInformation[]> {
+    const token = await this.systemToken(username)
+    const visitSchedulerApiClient = this.visitSchedulerApiClientBuilder(token)
+
+    logger.info(`Get booked and cancelled upcoming visits for ${offenderNo}`)
+    const visits = await visitSchedulerApiClient.getUpcomingVisitsCancelledAndBooked(offenderNo)
+
+    return visits.map(visit => {
```

Comment:
Can make this simpler 🙂 

`return visits.map(visit => this.buildVisitInformation(visit))`